src/main/jbake/content/docs/user/external-keycloak.adocPart

```diff
@@ -223,29 +219,15 @@ Remove from `$HAWKULAR_HOME/standalone/configuration/standalone.xml`:
 <property name="hawkular.events.listener.rest.endpoint" value="http://localhost:8080/hawkular-accounts-events-backend/events"/>
 ----
 
-Now, tell the Keycloak Wildfly Adapter what is the URL for the Keycloak Server:
-
-`$HAWKULAR_HOME/standalone/configuration/standalone.xml`:
+Change the property "keycloak.server.url" from `$HAWKULAR_HOME/standalone/configuration/standalone.xml` to point to
+our external Keycloak server URL:
```

Comment:
Nice catch! Will send another PR with the fix. To remove the ambiguity, I'll replace it by "the".